src/logging.conf

```diff
@@ -0,0 +1,36 @@
+[loggers]
+keys=root,uicheckapp
+
+[handlers]
+keys=consoleHandler,detailedConsoleHandler
+
+[formatters]
+keys=normalFormatter,detailedFormatter
+
+[logger_root]
+level=INFO
+handlers=consoleHandler
+
+[logger_uicheckapp]
```

Comment:
A note for myself...
You got this logging.conf file from a tutorial. Perhaps you should read up more on the Python logger and make sure everything in here is actually required. (specifically, what is `uicheckapp`?)

Reply:
I have removed unnecessary config from `logger.conf`